arelle/FunctionUtil.py

```diff
@@ -30,7 +30,7 @@ def stringArg(xc, args, i, type, missingArgFallback=None, emptyFallback=''):
     item = anytypeArg(xc, args, i, type, missingArgFallback)
     if item == (): return emptyFallback
     if isinstance(item, (ModelObject,ModelAttribute)):
-        return item.text
+        return item.text or ""
```

Comment:
I think `emptyFallback` is more appropriate here.

Reply:
Yes, sorry about that mistake, it's changed